volatility3/framework/plugins/windows/svclist.py

```diff
@@ -0,0 +1,102 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+
+from typing import List
+
+from volatility3.framework import interfaces, exceptions, symbols
+from volatility3.framework.configuration import requirements
+from volatility3.framework.symbols.windows import versions
+from volatility3.plugins.windows import svcscan, pslist
+from volatility3.framework.layers import scanners
+
+vollog = logging.getLogger(__name__)
+
+
+class SvcList(svcscan.SvcScan):
```

Comment:
We don't generally do inheritance, we do `VersionRequirement`s and then call class methods.  It's a bit more boilderplate, but making this explicit means stuff can't change out from under us later on...

Reply:
Made this to match the style of modscan/modules and the others and bumped versions and added requirements to match